static/styles/compose.css

```diff
@@ -222,6 +223,20 @@ table.compose_table {
     padding: 3px 8px 3px 8px;
 }
 
+#compose_private_stream_alert {
```

Comment:
Seems like these three changes are exact same as the invite css, `#compose_invite_users`. Maybe we just want an alert class or consolidate these two things together if possible.